1.0/profile.ttl

```diff
@@ -0,0 +1,63 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    dcterms:identifier "geosparql" ;
+    dcterms:created "2009-10-27"^^xsd:date ;
+    dcterms:modified "2012-07-19"^^xsd:date ;
+    dcterms:issued "2020-xx-xx"^^xsd:date ;
```

Comment:
I believe 2020-xx-xx is not as valid xsd:date. If only the year is known, why not use xsd:gYear?

Reply:
I was to update this when we actually present the Profile, so some time after this PR has gone in. Likely it will be 2021-xx-xx so I will indeed update it to "2021"^^xsd:gYear to keep it valid! 

I wanted this date separate from the created and modified, just as a testing of DCTERMS' properties.

Reply:
Updated to 2021 gYear